README.md

```diff
@@ -454,6 +454,37 @@ AS confederation parameters. Valid values are String, keyword 'default'.
 ##### `enforce_first_as`
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'.
 
+##### `event_history_cli`
+Enable/Disable cli event history buffer. Valid values are 'true', 'false', and 'default'.
+
+##### `event_history_cli_size`
+Set cli event history buffer size. Valid values are 'small', 'medium', 'large', 'disable' or 'default'.
```

Comment:
I see that the CLI behaves this way - what is not clear to me is what this actually means. I've searched internally and externally and I can't find anyone actually explaining what the operational difference is between `no event-history cli` and `event-history cli size disable`.

Another way to look at this - these two properties are heavily interdependent, as setting `cli_size` only works if `cli` is true (you can't set `no event-history cli` and `event-history cli size large` simultaneously). Additionally, there's a comment in the CLI suggesting that `event-history cli` is equivalent to `event-history cli size small`, and a quick manual test seems to bear this out:

```
n9k-108(config-router)# no event-history cli 
n9k-108(config-router)# show run bgp all | i cli
  no event-history cli
n9k-108(config-router)# event-history cli 
n9k-108(config-router)# show run bgp all | i cli
  event-history cli
n9k-108(config-router)# event-history cli size small
n9k-108(config-router)# show run bgp all | i cli
  event-history cli
n9k-108(config-router)# event-history cli size medium 
n9k-108(config-router)# show run bgp all | i cli
  event-history cli size medium
n9k-108(config-router)# event-history cli size small
n9k-108(config-router)# show run bgp all | i cli
  event-history cli
```

So it's not clear to me that there's any value in having this be split into two separate properties. You really only have four (*maybe* five) distinct states:

* disabled
* enabled, size small
* enabled, size medium
* enabled, size large
* enabled, size disabled (???)

I think it would really make more sense for this to be a single property.